src/odemis/util/dataio.py

```diff
@@ -120,7 +121,28 @@ def data_to_static_streams(data):
         elif dims in ("CYX", "YXC") and d.shape[ci] in (3, 4):
             # Only decide it's RGB as last resort, because most microscopy data is not RGB
             name = d.metadata.get(model.MD_DESCRIPTION, "RGB data")
-            klass = stream.RGBStream
+
+            # Check if it's a SEM or FIB image, and if so, use the SEMStream
+            if acqtype in [model.MD_AT_EM, model.MD_AT_FIB]:
+                if isinstance(d, model.DataArrayShadow):
+                    d = d.getData()
+
+                # get dim order, and select the first channel (arbitrary choice)
+                dims = d.metadata.get(model.MD_DIMS, "CYX")
+                if dims == "YXC":
```

Comment:
We shouldn't just automatically discard data. *only* do it if it's actually duplicate data (ie, greyscale image encoded as RGB). So you need to check using something like: `if dims == "YXC" and numpy.array_equal(d[:, :, 0] , d[:, :, 1]) and  numpy.array_equal(d[:, :, 0] , d[:, :, 2])`
(Maybe there is a nicer way to write, for instance using `ci`, but I couldn't really think of it quickly)

Reply:
The issue is that the images sometimes have rgb content (overlay, metadata icons) so they aren't identical channels, i've added an example + comment in the doc. 
I think it is pretty safe to do this, as the only way the MD_AT_EM tag gets added is if we parse it as an SEM image, the rgb content is an edge case of that for these exported images with overlays.